.github/workflows/upload.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2022 F4PGA Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier:ISC
+
+name: Upload FPGA interchange Data to GCP
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+
+  Upload-Interchange-Data:
+    runs-on: [self-hosted, Linux, X64]
+    container: ubuntu:focal
+
+    env:
+      GHA_SA: "gh-sa-interchange-tests-ci"
+      GCP_BUCKET: gs://fpga-interchange-tests/artifacts/prod/foss-fpga-tools/fpga-interchange-tests/continuous/install
+      DEBIAN_FRONTEND: noninteractive
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Install prerequisites
+      run: |
+        .github/scripts/ubuntu-setup.sh
+
+        echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+        curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+        apt update -qqy
+        apt install -qqy gnupg2 google-cloud-cli
+
+    - name: Timestamp
+      id: timestamp
+      run: |
+        TIMESTAMP="$(date +'%Y%m%d-%H%M%S')"
+        echo '::set-output name=timestamp::'"$TIMESTAMP"
```

Comment:
Instead of setting the output, you should be able to set an envvar for the following steps. The syntax is something similar to `echo 'VARNAME=VALUE' >> $GITHUB_ENV`. Yet, please do checks the docs as I wrote it by heart.

Reply:
Fixed